### PR TITLE
Release `taco@0.2.x`

### PR DESCRIPTION
**Pull request by piotr-roslaniec**
**Type of PR:**
- Other

**Required reviews:**
- 1

**Why it's needed:**
- One of the underlying dependencies, `@nucypher/nucypher-contracts` was updated
- Necessary to keep `taco-web` working for `tapir` users

### CODE REVIEW

**Review by manumonti — APPROVED** [review]
LGTM 👍

File: package.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
       "node-forge@<1.3.0": ">=1.3.0",
       "glob-parent@<5.1.2": ">=5.1.2",
       "nth-check@<2.0.1": ">=2.0.1",
-      "@nucypher/nucypher-core": "0.13.0-alpha.1"
+      "@nucypher/nucypher-core": "^0.14.1"
```
Comment (derekpierre) [inline package.json:85]: Judging by https://github.com/nucypher/nucypher-core/pull/89 there are no concerns about `0.14.1` not being compatible with nodes running `0.13.0` correct? The breaking changes were limited to the wasm bindings, which the nodes don't care about.

File: packages/taco/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nucypher/taco",
-  "version": "0.1.0",
+  "version": "0.2.2",
```
Comment (derekpierre) [inline packages/taco/package.json:3]: Why 0.2.2 and not 0.2.0?

File: package.json (hunk 1 above)
Comment (piotr-roslaniec) [reply package.json:85]: Yes, there changes are strictly for `nucypher-core-wasm` and there is no impact on Python users.

File: packages/taco/package.json (hunk 2 above)
Comment (piotr-roslaniec) [reply packages/taco/package.json:3]: I messed up and published a bad build. I yanked previous versions, but `npmjs.com` doesn't allow republishing so I had to bump the version.

**Pull request merged**